src/V3Os.h

```diff
@@ -58,6 +59,9 @@ class V3Os {
     static void u_sleep(int64_t usec);  ///< Sleep for a given number of microseconds.
     static uint64_t timeUsecs();  ///< Return wall time since epoch in microseconds, or 0 if not implemented
     static uint64_t memUsageBytes();  ///< Return memory usage in bytes, or 0 if not implemented
+
+    // METHODS (sub command)
+    static int system(const string& command);  ///< Run system command, returns the exit code of the child process.
```

Comment:
```suggestion
    /// Run system command, returns the exit code of the child process.
    static int system(const string& command);
```

(Otherwise line too long)